spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/Fabric8ConfigMapPropertySource.java

```diff
@@ -41,25 +44,21 @@
 	private static final Log LOG = LogFactory.getLog(Fabric8ConfigMapPropertySource.class);
 
 	public Fabric8ConfigMapPropertySource(KubernetesClient client, String name) {
-		this(client, name, null, (Environment) null);
+		this(client, name, null, null);
 	}
 
-	public Fabric8ConfigMapPropertySource(KubernetesClient client, String name, String namespace,
+	public Fabric8ConfigMapPropertySource(KubernetesClient client, String applicationName, String namespace,
 			Environment environment) {
-		super(getName(name, getNamespace(client, namespace)),
-				asObjectMap(getData(client, name, getNamespace(client, namespace), environment)));
-	}
-
-	private static String getNamespace(KubernetesClient client, String namespace) {
```

Comment:
was not really removed, but moved to `Fabric8ConfigUtils`. Might come handy in future other places too; besides this surely looks like a "utility" method, so why not place it in the proper place?